samples/24.bot-authentication-msgraph/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Remove this from PR.  We might need to add to gitignore.

Reply:
Removed the settings file and it's in the .gitignore, not sure why it's not honoring it. I'll check into it further. Also changed the package name so it's not the same as the other sample, just in case it might cause confusion.